modules/serverless-rn-1-6-0.adoc

```diff
@@ -0,0 +1,71 @@
+* Parts of the `KnativeServing` control plane and data plane are now deployed as highly available (HA) by default.
+* Kourier is now actively watched and reconciles changes automatically.
+* {ServerlessProductName} now supports use on {product-title} nightly builds.
+
+[id="fixed-issues-1-6-0_{context}"]
+== Fixed issues
+* In previous versions, the `oc explain` command did not work correctly. The structural schema of the `KnativeServing` CRD has been updated in {ServerlessProductName} 1.6.0 so that the `oc explain` command now works correctly.
+* In previous versions, it was possible to create more than one `KnativeServing` CR. Multiple `KnativeServing` CRs are now prevented synchronously in {ServerlessProductName} 1.6.0. Attempting to create more than one `KnativeServing` CR now results in an error.
+* In previous versions, {ServerlessProductName} was not compatible with {product-title} deployments on GCP. This issue has been fixed in {ServerlessProductName} 1.6.0.
+// * In previous versions,  requests for `KnativeServing` CRs without specifying an API group, for example, `oc get knativeserving -n knative-serving`, would occasionally cause errors. This issue has been fixed in {ServerlessProductName} 1.6.0.
+// Move this fixed issue to 1.7.0 release notes once these are created and it's confirmed as fixed
+* In previous releases, the Knative Serving webhook would crash with an out of memory error if the cluster had more than 170 namespaces. This issue has been fixed in {ServerlessProductName} 1.6.0.
+* In previous releases, {ServerlessProductName} did not automatically fix an {product-title} route that it created if the route was changed by another component. This issue has been fixed in {ServerlessProductName} 1.6.0.
+* In previous versions, deleting a `KnativeServing` CR would occasionally cause the system to hang forever. This issue has been fixed in {ServerlessProductName} 1.6.0.
+* Due to the ingress migration from Service Mesh to Kourier that occured in {ServerlessProductName} 1.5.0, orphaned VirtualServices were able to potentially remain on the system. In {ServerlessProductName} 1.6.0, orphaned VirtualServices are automatically removed.
+
+[id="known-issues-1-6-0_{context}"]
+== Known issues
+* In {ServerlessProductName} 1.6.0, if a cluster administrator uninstalls {ServerlessProductName} using the uninstall procedure provided in the documentation, the *Serverless* dropdown will still be visible in the *Administrator* perspective of the {product-title} web console, and the *Knative Service* resource will still be visible in the *Developer* perspective of the {product-title} web console.
+A user will still be able to create Knative services using this option, however these Knative services will no longer work for the deployment.
++
+To prevent {ServerlessProductName} from being visible in the {product-title} web console, the cluster administrator must delete additional CRDs from the deployment after removing the Knative Serving CR.
```

Comment:
Is this described in the relevant assembly, too?

Reply:
It will be - I have a separate issue for this as mentioned above.